Modified the Trader.java, and create a new class called Sheep.java wh…

johnnyjly opened the pull request:
…ich implemented all the interfaces provided.

Also, there is a SheepTest.java (contains a small error).

johnnyjly2 committed:
Modified the Trader.java, and create a new class called Sheep.java which implemented all the interfaces provided.
Also, there is a SheepTest.java (contains a small error).

hmartyscu commented on src/test/java/SheepTest.java:
```diff
@@ -0,0 +1,34 @@
+        s.upgradeSpeed();
+        assertEquals(3, s.getMaxSpeed());
+    }
+
```
Missing test case for upgradeSpeed

hmartyscu commented on src/main/java/Sheep.java:
```diff
@@ -0,0 +1,39 @@
+public class Sheep implements Drivable, Domesticatable, Tradable{
```
Code style: there should be a space before the brace

hmartyscu reviewed (changes requested):
I've found two problems so far

johnnyjly2 committed:
Modified the Sheep.java and SheepTest.java.
For Sheep.java, add a space before the curly brace.
For SheepTest.java, add a test case.
